app/helpers/pipeline_outputs_helper.rb

```diff
@@ -140,6 +140,18 @@ def taxon_name(taxid, tax_level)
     taxon["#{tax_level}_name"] if taxon
   end
 
+  def get_taxid_fasta_from_pipeline_run_combined_nt_nr(pipeline_run, taxid, tax_level)
```

Comment:
`get_taxid_fasta` sounds off to me... isn't the fasta a property of the taxon, not the taxid?

Reply:
Makes sense. Thanks.